examples/typescript/lnd/tsconfig.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
this is largely the defaults from `tsc init`.

I'll clean it up and remove the comments in the 90% version of the PR.

Reply:
I think it's fine to leave the comments. It's likely helpful for devs new to Typescript.